### PR TITLE
[AG-1385] Fixes JSON Schema expectation passing when it should fail

### PR DESCRIPTION
**Pull request by BWMac**
**Problem:**

The `expect_column_values_to_match_json_schema` expectation was not correctly failing when members of the `go_terms` list in the `gene_biodomains` field were not all unique. After investigation, I determined that this was due to the `uniqueItems` JSON Schema property not being applied correctly in our `gene_biodomains` JSON Schema file.

**Solution:**

Fix the application of the property so that it works as intended.

**Testing:**

In order to ensure that the expectation and therefore the data pipeline is failing when we expect, I created an invalid `genes_biodomains.json` file with non-unique `go_terms` and ran the pipeline with these changes applied. The processing error was raised as expected.

Bad JSON I used for testing:
```
{
    "ensembl_gene_id": "ENSG00000000003",
    "gene_biodomains": [
      {
        "biodomain": "Apoptosis",
        "go_terms": [
          "positive regulation of canonical NF-kappaB signal transduction",
          "negative regulation of non-canonical NF-kappaB signal transduction",
          "negative regulation of non-canonical NF-kappaB signal transduction" // Repeated `go_term`
        ],
        "n_biodomain_terms": 194,
        "n_gene_biodomain_terms": 2,
        "pct_linking_terms": 100.0
      }
    ]
  }
```

Error received:
```
agoradatatools.errors.ADTDataProcessingError: 
Data Processing has failed for one or more data sources. Refer to the list of errors below to address issues:
genes_biodomains: Great Expectations data validation has failed: gene_biodomains has failed expectations expect_column_values_to_match_json_schema
```

### CODE REVIEW

File: src/agoradatatools/great_expectations/gx/expectations/genes_biodomains.json [hunk 1]
```diff
@@ -94,12 +94,12 @@
...
                 "default": [],
                 "items": {
-                  "UniqueItems": true,
                   "type": "string"
                 },
                 "maxItems": 100,
                 "minItems": 1,
-                "type": "array"
+                "type": "array",
+                "uniqueItems": true
```
Comment (thomasyu888) [inline src/agoradatatools/great_expectations/gx/expectations/genes_biodomains.json:102]: 👀 tricky...

**Review by thomasyu888 — APPROVED** [review]
🔥 LGTM!

**Review by jaclynbeck-sage — APPROVED** [review]
Good catch! Looks good to me.

**Review by JessterB — APPROVED** [review]
Sometimes it's the little things... ;)

**Pull request merged**